src/store/getters/state-getters.ts

```diff
@@ -218,14 +218,26 @@ export const isIncorporationAgreementValid = (state: any): boolean => {
 }
 
 /** Whether all the incorporation steps are valid. */
-export const isApplicationValid = (state: any): boolean => {
-  return (
-    state.stateModel.defineCompanyStep.valid &&
-    state.stateModel.addPeopleAndRoleStep.valid &&
+export const isApplicationValid = (state: any, getters: any): boolean => {
+  // Base company steps
+  const isCompanyStepsValid = (
```

Comment:
I like the comment, but the variable name had me going "hmm" for a bit. Suggest you rename it isBaseStepsValid.